### DISCUSSION TITLE
Acer RGB Menu for this module

### DISCUSSION THREAD

**Discussion by Zeaksblog** [JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module#164] [Show and tell]
I made a simple menu to make it easier to control the module. You can select from all effect modes, for the static mode you can choose between zones or all zones, create new profile, list and load profiles, at the end of creating a new effect it will ask if you want to save it as a new profile. Depending on the effect you can select speed, I'll add brightness and left-right right-left soon and anything else that's missing

You can get it here https://github.com/Zeaksblog/acer-rgb-menu

The model of my laptop isn't listed but the RGB module works great Predator PHN16-71

**Comment by JafarAkhondali** [comment]
Hey @Zeaksblog 
Thanks for sharing 👍🏻 You can also send a pull request to add your model to readme
